src/main/java/Penguin.java

```diff
@@ -0,0 +1,238 @@
+import java.io.IOException;
+import java.sql.SQLOutput;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+
+public class Penguin {
+    public static void main(String[] args) {
+        String name = "Penguin";
```

Comment:
Can consider declaring as static final
```suggestion
        public static final String name = "Penguin";
```